lib/rubocop/cop/bundler/ordered_gems.rb

```diff
@@ -17,7 +17,8 @@ module Bundler
       #
       #   gem 'rspec'
       class OrderedGems < Cop
-        MSG = 'Gem `%s` should appear before `%s` in their gem group.'.freeze
+        MSG = 'Alphabetize gems: Gem `%s` should appear before `%s` '\
```

Comment:
should we keep the names of gems (like `x` should appear before `y`) or not required?